src/Native/SymSgdNative/CMakeLists.txt

```diff
@@ -18,4 +18,8 @@ endif()
 add_library(SymSgdNative SHARED ${SOURCES} ${RESOURCES})
 target_link_libraries(SymSgdNative PUBLIC ${MKL_LIBRARY})
 
+if(APPLE)
+    set_target_properties(SymSgdNative PROPERTIES INSTALL_RPATH "@loader_path;@loader_path/../../../../../${MKL_LIB_RPATH};${MKL_LIB_PATH}")
```

Comment:
>@loader_path/../../../../../${MKL_LIB_RPATH} [](start = 78, length = 44)

How about let's put the `../../..` outside of the CMake files as well? The whole relative path should be passed into `${MKL_LIB_RPATH}`, so here we can just say:

`@loader_path/${MKL_LIB_RPATH}`